projects/kit/src/lib/masks/number/plugins/leading-zeroes-validation.plugin.ts

```diff
@@ -10,25 +11,42 @@ const DUMMY_SELECTION = [0, 0] as const;
  * @example 000000 => blur => 0
  * @example 00005 => blur => 5
  */
-export function createLeadingZeroesValidationPlugin(
-    decimalSeparator: string,
-    thousandSeparator: string,
-): MaskitoPlugin {
-    const dropRepeatedLeadingZeroes = createLeadingZeroesValidationPostprocessor(
+export function createLeadingZeroesValidationPlugin({
+    decimalSeparator,
+    thousandSeparator,
+    prefix,
+    postfix,
+}: {
+    decimalSeparator: string;
+    thousandSeparator: string;
+    prefix: string;
+    postfix: string;
+}): MaskitoPlugin {
+    const dropRepeatedLeadingZeroes = createLeadingZeroesValidationPostprocessor({
         decimalSeparator,
         thousandSeparator,
-    );
+        prefix,
+        postfix,
+    });
 
     return maskitoEventHandler(
         'blur',
         element => {
-            const newValue = dropRepeatedLeadingZeroes(
-                {
-                    value: element.value,
-                    selection: DUMMY_SELECTION,
-                },
-                {value: '', selection: DUMMY_SELECTION},
-            ).value;
+            const {cleanValue, extractedPostfix, extractedPrefix} = extractAffixes(
```

Comment:
`createLeadingZeroesValidationPostprocessor` can handle cases with prefix/postfix now. Should we drop these lines now ?